release-1.9/release-1.9.md

```diff
@@ -0,0 +1,89 @@
+# Proposed timeline for 1.9
+We will begin the 1.9 release the first week of October 2017.
+
+The proposal below is similar in layout previous releases:
+- as before, key days aren't Fridays, since it can be hard to end milestones right up against weekends
+- there are a number of North American holidays during Q4
+- same as in 1.8
+  * alpha's should not be cut unless all master-release-blocking tests are passing
+  * release notes for planned features done 2 weeks out from feature freeze
+  * docs PRs deadlines ahead of release
+
+## 1.9 Release Schedule Key Dates
+- Oct 5nd (Weds) coding start
+- Oct 11th (Weds): v1.9.0-alpha.1
+- Oct 25th (Weds): v1.9.0-alpha.2
+- Oct 27th (Fri): Feature Freeze
+- Nov 8th (Weds): v1.9.0-alpha.3
+- Nov 22nd (Weds): v1.9.0-alpha.4
+- Nov 22nd (Weds): Code Freeze 
```

Comment:
Between freeze and ga is thanksgiving and kubecon. Both will limit availability of people to resolve issues.

Reply:
Also, immediately following code freeze there is typically a submit queue backlog of approved/unmerged PRs that require broad babysitting/rebasing. That doesn’t align well with a four-day weekend immediately following in the US

Reply:
This seems tight, is there any possibility of releasing in 2018?

Reply:
At the very least, we should end code freeze on Friday, December 8th.

Reply:
@jdumars why the 8th vs the 6th?  Longer dev period?